tools/itch5bookdepth.cpp

```diff
@@ -0,0 +1,146 @@
+#include <jb/itch5/compute_book_depth.hpp>
+#include <jb/itch5/process_iostream.hpp>
+#include <jb/book_depth_statistics.hpp>
+#include <jb/fileio.hpp>
+#include <jb/log.hpp>
+
+#include <iostream>
+#include <stdexcept>
+#include <unordered_map>
+
+#include <chrono>
+
+/* Ticket https://github.com/coryan/jaybeams/issues/20
+ *
+ * See https://github.com/GFariasR/jaybeams/wiki/ITCH5-Depth-of-Book-StatisticsProject
```

Comment:
One link to the overall design is great though...  good idea, will use in the future.